java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -3664,4 +3664,69 @@ public void bitcount(BaseClient client) {
             assertTrue(executionException.getCause() instanceof RequestException);
         }
     }
+
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void bitpos(BaseClient client) {
+        String key1 = UUID.randomUUID().toString();
+        String key2 = UUID.randomUUID().toString();
+        String key3 = UUID.randomUUID().toString();
+        String value = "?f0obar"; // 00111111 01100110 00110000 01101111 01100010 01100001 01110010
```

Comment:
I, again, feel like using byte[] here might be more obvious.   That way, 0x010 we know has a 1 in the second position.

Reply:
I realize we might have to go back and clean up your previous commands to fix this...

Reply:
As per the discussion offline, we are going to use long. This can be switched in the future if a boolean or byte is preferred.